pkg/types/types.go

```diff
@@ -201,23 +230,29 @@ type HostDevice interface {
 	GetDeviceCode() string
 }
 
-// PciDevice provides an interface to get PCI device specific information
+// PciDevice provides an interface to get generic PCI device information
 // extends HostDevice interface
 type PciDevice interface {
 	HostDevice
 	GetPciAddr() string
+}
+
+// NetDevice provides an interface to get generic network device information
+type NetDevice interface {
+	HostDevice
+	GetPFName() string
```

Comment:
can you rename to `GetPFNetName` ? it seems more suitable.

Reply:
Should it be `GetPFNetName` or `GetPfNetName` as `GetPfPciAddr`? Or make `PF` everywhere?

Reply:
+1 with being consistent. i dont have a special preference.